crown-court-proceeding/src/test/java/uk/gov/justice/laa/crime/crowncourt/controller/CrownCourtProceedingIntegrationTest.java

```diff
@@ -202,6 +207,78 @@ void givenAValidContent_whenUpdateApplicationIsInvoked_thenUpdateApplicationIsSu
         assertThat(result.getResponse().getContentAsString()).isEqualTo(objectMapper.writeValueAsString(updateApplicationResponse));
     }
 
+
+    @Test
+    void givenAEmptyOAuthToken_whenUpdateIsInvoked_thenFailsUnauthorizedAccess() throws Exception {
+        mvc.perform(RequestBuilderUtils.buildRequestGivenContent(
+                        HttpMethod.PUT, "{}", ENDPOINT_URL + UPDATE_CC_URL, Boolean.FALSE))
+                .andExpect(status().isUnauthorized());
+    }
+
+    @Test
+    void givenAInvalidContent_whenUpdateIsInvoked_thenFailsBadRequest() throws Exception {
+        mvc.perform(RequestBuilderUtils.buildRequestGivenContent(
+                        HttpMethod.PUT, objectMapper.writeValueAsString(
+                                TestModelDataBuilder.getApiUpdateApplicationRequest(Boolean.FALSE)), ENDPOINT_URL + UPDATE_CC_URL))
+                .andExpect(status().isBadRequest());
+    }
+
+    @Test
+    void givenAValidContent_whenUpdateIsInvoked_thenUpdateIsSuccess() throws Exception {
```

Comment:
This constitutes the 'happy path', but what about the not so happy path? The other tests mirror what we have in the controller, so don't seem to be adding much. Personally, I'd like to see at least one more scenario covering for example an internal validation failure and/or a failing call to a downstream service.